### PR TITLE
refactor(condo): decrease useless number of utility files

### PR DESCRIPTION
**Pull request by pahaz**

### CODE REVIEW

**Comment by sitozzz** [comment]
There are more declaration of this function in the code. For example, `apps/condo/domains/user/components/containers/AuthLayoutContext.tsx`. Should it be deleted too?

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 1]
```diff
@@ -16,6 +15,13 @@ interface ILayoutContext {
     addNotification?: (notification: ITopNotification) => void
 }
 
+const detectMobileNavigator = (): boolean => {
```
Comment (AntonAL) [inline apps/condo/domains/common/components/LayoutContext.tsx:18]:
```suggestion
const detectMobileUserAgent = (): boolean => {
```

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/common/components/LayoutContext.tsx (hunk 1 above)
Comment (akarjakin) [inline apps/condo/domains/common/components/LayoutContext.tsx:18]: we have two places where this function is used, other one is at `apps/condo/domains/user/components/containers/AuthLayoutContext.tsx`

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 2]
```diff
@@ -16,6 +15,13 @@ interface ILayoutContext {
...
+    return (
+        typeof window !== 'undefined'
+        && /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(window.navigator.userAgent)
```
Comment (akarjakin) [inline apps/condo/domains/common/components/LayoutContext.tsx:21]: for optimization sake it's better to move this static RegExp to a constant, in order to skip extra compilations in runtime

File: apps/condo/domains/common/components/LayoutContext.tsx (hunk 1 above)
Comment (pahaz) [reply apps/condo/domains/common/components/LayoutContext.tsx]: Yes. I've fix one. Another is not a part of condo app

**Comment by pahaz** [comment]
So. We also have the `detectMobileNavigator` in miniapp. But there is another app.

**Review by sitozzz — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 3]
```diff
@@ -16,6 +15,13 @@ interface ILayoutContext {
     addNotification?: (notification: ITopNotification) => void
 }
 
+const isMobileUserAgent = (): boolean => {
+    return (
+        typeof window !== 'undefined'
+        && /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(window.navigator.userAgent)
```
Comment (akarjakin) [inline apps/condo/domains/common/components/LayoutContext.tsx:21]: better to move regexp to a constant to skip extra compilations in runtime

**Pull request merged**